CHANGELOG.md

```diff
@@ -9,16 +9,31 @@ All notable changes to AET will be documented in this file.
 
 ## Unreleased
 **List of changes that are finished but not yet released in any final version.**
-- [PR-41](https://github.com/Cognifide/aet/pull/41) - [Click](https://github.com/Cognifide/aet/wiki/ClickModifier)
- and [Hide](https://github.com/Cognifide/aet/wiki/HideModifier) modifiers allows to use css selectors as parameters. 
- Added [Replace Text](https://github.com/Cognifide/aet/wiki/ReplaceTextModifier) modifier and it's tests.
-- [PR-39](https://github.com/Cognifide/aet/pull/39) - fixed filtering w3cHtml issues.
+- 
 
+## Version 2.0.2
```

Comment:
We should put here only significant changes, PRs that do not fix/extend/introduce a new functionality should not be mentioned here (e.g. fixed Unit Test).